substrate/client/network/src/request_responses.rs

```diff
@@ -318,7 +318,6 @@ impl RequestResponsesBehaviour {
 		let mut protocols = HashMap::new();
 		for protocol in list {
 			let mut cfg = Config::default();
-			cfg.set_connection_keep_alive(Duration::from_secs(10));
```

Comment:
dq: Do we handle this elsewhere?

Reply:
Yes, it moved to the global `Swarm` config: https://github.com/paritytech/polkadot-sdk/blob/6e5b4effa1fe4d31b1eca6e9fb643acb3e222fbe/substrate/client/network/src/service.rs#L587